fermenter-mda/pom.xml

```diff
@@ -189,11 +207,29 @@
                     </execution>
                 </executions>
             </plugin>
-            <!-- Since we have cucumber surefire tests that run as part of the build,
-                generate cucumber reports -->
             <plugin>
                 <groupId>net.masterthought</groupId>
                 <artifactId>maven-cucumber-reporting</artifactId>
+                <version>5.8.1</version>
+                <executions>
+                    <execution>
+                        <id>execution</id>
+                        <phase>post-integration-test</phase>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <projectName>fermenter</projectName>
+                            <outputDirectory>${project.build.directory}/cucumber-html-reports</outputDirectory>
+                            <inputDirectory>${project.build.directory}/../target/cucumber-reports</inputDirectory>
```

Comment:
A: flatten file path for improved clarity (we sometimes use this format for python modules where the build directory is `dist`, but that is not the case in Fermenter, so make it more direct)